### PR TITLE
Doc/#1495 -  Add document for filtering scenarios with creation_time

### LINKED ISSUE #1043: [DOCS] Document the Time filtering for scenario getters
### Issue Description

Documentation for the feature from https://github.com/Avaiga/taipy/issues/393

### Screenshots or Examples (if applicable)

_No response_

### Proposed Solution (optional)

_No response_

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by trgiangdo**
Resolves #1043

### CODE REVIEW

File: docs/manuals/userman/sdm/scenario/index.md [hunk 1]
```diff
@@ -288,6 +289,8 @@ returns the list of all existing scenarios. Here is the list of the parameters:
     order for dates, in alphabetical order for name and id, and in lexicographical order for tags.
     The default value is "name". If an incorrect sorting key is provided, the scenarios are sorted
     by name.
+- *created_start_time* represents the optional inclusive start date to filter scenarios by creation date.
+- *created_end_time* represents the optional inclusive end date to filter scenarios by creation date.
```
Comment (jrobinAV) [inline docs/manuals/userman/sdm/scenario/index.md]: Hum.

Usually, working on intervals, we include the start and **exclude** the end: `[start; end[`.
This is more convenient when it comes to use the API iteratively in a loop. So an iteration and the following are disjoint.

In other words, the two following calls can be safely treated independently.
```
taipy.get_scenarios(start, intermediate)
taipy.get_scenarios(intermediate, end)
```

File: docs/manuals/userman/sdm/scenario/index.md [hunk 2]
```diff
@@ -317,6 +320,8 @@ existing cycles. The list of primary scenarios can be sorted using the parameter
...
```
Comment (jrobinAV) [inline docs/manuals/userman/sdm/scenario/index.md]:
```suggestion
- *created_end_time* represents the optional exclusive end date to filter scenarios by creation date.
```

File: docs/manuals/userman/sdm/scenario/index.md (hunk 1 above)
Comment (trgiangdo) [reply docs/manuals/userman/sdm/scenario/index.md]: I understand. However, this would require similar changes on taipy. Should I handle that as well?

Comment (jrobinAV) [reply docs/manuals/userman/sdm/scenario/index.md]: Yep, please. Sorry about that. I missed it during the code review.

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**